src/git/commit_filter.go

```diff
@@ -111,7 +119,7 @@ func (s *CommitFilter) Commits(lastHash string) ([]Commit, error) {
 
 	filteredCommits := make([]Commit, 0)
 	for _, commit := range commits {
-		if !s.commitChangesExcluded(commit.Files) {
+		if !s.commitChangesExcluded(commit.Files) && !s.commitDevDependenciesExcluded(commit.Message) {
```

Comment:
nit: I wonder if, now that we have more than one filter, we should rename `commitChangesExcluded` to `commitExcludedByFiles` (just to make each function responsibility more explicit)

Reply:
Yes since the `commitChangesExcluded ` it's generic to excluded changed; thanks for the suggestion 👍

Reply:
Done in 1c488f32c1c81e2107b13ffb8801c864bbefaa8e.